packages/@expo/config-types/src/ExpoConfig.ts

```diff
@@ -246,6 +246,10 @@ export interface ExpoConfig {
      * If true, indicates that this project does not support tablets or handsets, and only supports Apple TV and Android TV
      */
     supportsTVOnly?: boolean;
+    /**
+     * If true, the window spans the entire display size by drawing behind transparent system bars.
+     */
+    edgeToEdge?: boolean;
```

Comment:
This file is auto-generated from a jsonschema in another (private) repo. The change needs to be made upstream there and then sync'ed with this repo (see the readme of `@expo/config-types` on how to sync). Not sure if you have access, though, in which case we can decide on the naming and comment in this PR and then I can make the upstream PR to add the field for you.

Reply:
@wschurman - I will take care of this when needed! I'm working with @zoontek on this